### DISCUSSION TITLE
How to set scriptEvaluationTimeout for remote connection

### DISCUSSION THREAD

**Discussion by pallakartheekreddy** [JanusGraph/janusgraph#4487] [Q&A] [answered]
We are attempting to increase the `scriptEvaluationTimeout` in the `remote-object.yaml` file by setting the value below. However, this change is not taking effect, and the timeout remains at 30000 (30 seconds).
```gremlin.remote.request.timeout=60000```

Any suggestion how to fix this issue?

**Comment by hadoopmarc** [comment, marked as answer]
See evaluationTimeout in https://tinkerpop.apache.org/docs/current/reference/#_configuring_2
